terminatorlib/terminal.py

```diff
@@ -1018,6 +1018,9 @@ def on_mousewheel(self, widget, event):
             elif event.direction == Gdk.ScrollDirection.DOWN or SMOOTH_SCROLL_DOWN:
                 self.scroll_by_page(1)
                 return (True)
+        if event.state & Gdk.ModifierType.MOD1_MASK == Gdk.ModifierType.MOD1_MASK:
```

Comment:
`Gdk.ModifierType.MOD1_MASK == Gdk.ModifierType.MOD1_MASK` would always evaluate to true so it can be removed from this conditional

Reply:
ok, sure.  I was copying the coding style of the other event.state conditionals near by.

Reply:
Changing it to `!= 0` would be better. Also, I just notice it, please use `and` instead of `&` - it's more pythonic. So the full conditional could be just `if event.state and Gdk.ModifierType.MOD1_MASK != 0`.

Reply:
Is 'and' bitwise ?  Not sure how it can be both bitwise and logical ?  I think its pretty common to use '&' for bitwise ?
Would parens help ?
Again I'm just following the same style as several other lines above it.

Reply:
Ok, my mistake. Maybe just leave as it is since other lines are in the same style. Sorry for the confusion.